change-beta/@azure-communication-react-620cfff7-2075-4461-b442-1363c891c420.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "prerelease",
+  "area": "improvement",
+  "workstream": "removing unwanted dependency",
```

Comment:
`workstream` is the feature workstream, in this case it would be `reactions`. Its just used for grouping like changes in the changelog generation

Reply:
I have been using work stream in the wrong way this whole time ⭕